src/neo/SmartContract/ApplicationEngine.Storage.cs

```diff
@@ -101,9 +101,9 @@ private void PutExInternal(StorageContext context, byte[] key, byte[] value, Sto
             {
                 if (item.IsConstant) throw new InvalidOperationException();
                 if (value.Length <= item.Value.Length)
-                    newDataSize = 1;
+                    newDataSize = (1 + value.Length / 4);
```

Comment:
Why `+1`? I think if it is `0` then we consider it as `1`.

Reply:
It's ok for me, I'll adjust it.

Reply:
Done